JASP-Engine/JASP/R/descriptives.R

```diff
@@ -1246,8 +1248,6 @@ Descriptives <- function(jaspResults, dataset, options) {
     p<-NULL
   }
   descriptivesQQPlot <- createJaspPlot(plot=p, width=400, aspectRatio=1, title=title)
-  if (!is.null(levelName))
-    descriptivesQQPlot$dependOn(optionContainsValue=list(variables=qqvar))
```

Comment:
You can't remove this, because otherwise no dependencies are set when you do not have a grouping variable.

Reply:
Although it should totally be 
`if (is.null(levelName))` rather than  `if (!is.null(levelName))`